content/introduction/why-dolt.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Why Dolt
+---
+
+## Background
+Existing relational database solutions came to market to provide a persistence layer for the multi-user applications the internet enabled.
```

Comment:
I don't think this sentence is even close to true. Relational databases existed before the internet. What is the point you are trying to make?

Reply:
I mean... partially. Oracle was released in 1979, so that point internal networks existed such that there were multi user applications. There is really no substantial relational database market without multi-user applications. The theory of relational data comes from the 50s, but adoption really takes off with the internet and the new class of corporate IT tools that comes along with it.

I feel that my pithy remark approximates this?

Reply:
If it were pithy, I'd give it a pass. 

Existing relational database were built for an online use case.
Existing relational databases were built in a different era of computing. 
Existing relational databases were built to serve online applications.

All good. The problem is that you were too specific to be considered hand wavy I think.

Reply:
if you don't like online, you can also go with networked.